MonkeyWrench.Web.WebService/Scheduler/Scheduler.cs

```diff
@@ -317,13 +317,14 @@ private static void AddWork (DB db, List<DBHost> hosts, List<DBLane> lanes, List
 		{
 			DateTime start = DateTime.Now;
 			List<DBCommand> commands = null;
-			List<DBLaneDependency> dependencies = null;
+			List<DBLaneDependency> dependencies = DBLaneDependency_Extensions.GetDependencies (db, null);

```

Comment:
Ah, I had convinced myself that there wouldn't be a case that they would never not be fetched, but I forgot that this method is called even when there are no new revisionworks.